test/unit/serialization/APortable.js

```diff
@@ -14,9 +14,8 @@
  * limitations under the License.
  */
 'use strict';
-
 class APortable {
```

Comment:
You need to add arrays of these to `aPortable`. You will exclude the arrays only from reference objects.